### PR TITLE
[Feature] Complement student information for teacher section

### PR DESCRIPTION
**Pull request by bryanmg**
## Description
- Teacher cannot see or edit the learning resources already signed to student or reviews made.

## Solution
- Add history of reviews made and signed learning resources to student

## Screenshots
Screen Shot 2022-03-04 at 14 01 32

### CODE REVIEW

File: app/controllers/student_learning_resources_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class StudentLearningResourcesController < ApplicationController
   before_action :authenticate_user!, :set_user
+  before_action :set_student_learning_resource, only: [:destroy]
```
Comment (itsmeurbi) [inline app/controllers/student_learning_resources_controller.rb:3]: I don't see the need for a one-line before action only used for one single method

File: app/controllers/student_learning_resources_controller.rb [hunk 2]
```diff
@@ -17,8 +18,18 @@ def create
     end
   end
 
+  def destroy
+    student_id = @student_learning_resource[:user_id]
```
Comment (itsmeurbi) [inline app/controllers/student_learning_resources_controller.rb:22]: I don't see the need of treating `@student_learning_resource` as a hash 
If this is an object then you better send it a message 
```
student_id = @student_learning_resource.student_id
```

File: app/controllers/teachers/reviews_controller.rb [hunk 3]
```diff
@@ -29,8 +29,9 @@ def update
     end
 
     def destroy
+      student_id = @review[:student_id]
```
Comment (itsmeurbi) [inline app/controllers/teachers/reviews_controller.rb:32]: Here as well. Why do you treat `@review` as a hash?

File: app/views/teachers/students/show.html.erb [hunk 4]
```diff
@@ -35,6 +35,68 @@
       </tbody>
     </table>
   </div>
+
+  <%# Student record of learning resources %>
+  <h2 class="subtitle is-flex is-justify-content-center">
+    <strong>Asigned learning resources</strong>
```
Comment (itsmeurbi) [inline app/views/teachers/students/show.html.erb:41]: Should be `Assigned`

File: app/views/teachers/students/show.html.erb [hunk 5]
```diff
@@ -35,6 +35,68 @@
...
+  </h2>
+  <div class="container mb-5">
+    <table class="table is-bordered  is-narrow is-hoverable is-fullwidth">
+      <thead>
+        <tr class="has-background-primary-light">
+          <th>Learnign resource name</th>
+          <th>Asigned on</th>
```
Comment (itsmeurbi) [inline app/views/teachers/students/show.html.erb:48]: Should be `Assigned to`

File: app/views/teachers/students/show.html.erb [hunk 6]
```diff
@@ -35,6 +35,68 @@
...
+      </thead>
+      <tbody>
+        <% @student.student_learning_resources.each do |resource| %>
+          <tr>
+            <td><%= resource.learning_resource.name %></td>
+            <td><%= resource.learning_resource.created_at.strftime('%Y-%m-%d %T') %></td>
+            <td class="is-flex is-justify-content-center">
+              <%= button_to(teacher_student_learning_resource_path(@user, resource), class: "button is-info is-outlined is-small ", method: :delete, data: { confirm: 'Are you sure to want to unasigne this resource?' }) do %>
+                <i class="far fa-trash-alt"></i>
+                <span class="ml-2">Unasigne</span>
```
Comment (itsmeurbi) [inline app/views/teachers/students/show.html.erb:60]: Should be `Unassign`

File: app/views/teachers/students/show.html.erb [hunk 7]
```diff
@@ -35,6 +35,68 @@
...
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+
+  <%# Student reviews made %>
+  <h2 class="subtitle is-flex is-justify-content-center">
+    <strong>Reviews to the student</strong>
```
Comment (itsmeurbi) [inline app/views/teachers/students/show.html.erb:71]: I'd say this should be `Student's review`

**Review by itsmeurbi — CHANGES_REQUESTED** [review]

File: app/controllers/student_learning_resources_controller.rb (hunk 1 above)
Comment (bryanmg) [reply app/controllers/student_learning_resources_controller.rb:3]: I supposed it was a god practice.

File: app/controllers/student_learning_resources_controller.rb (hunk 2 above)
Comment (bryanmg) [reply app/controllers/student_learning_resources_controller.rb:22]: Actually, that line is not required

File: app/controllers/teachers/reviews_controller.rb (hunk 3 above)
Comment (bryanmg) [reply app/controllers/teachers/reviews_controller.rb:32]: Same here, this line is not required

File: app/views/teachers/students/show.html.erb (hunk 5 above)
Comment (bryanmg) [reply app/views/teachers/students/show.html.erb:48]: my bad, typo

File: app/views/teachers/students/show.html.erb (hunk 6 above)
Comment (bryanmg) [reply app/views/teachers/students/show.html.erb:60]: tanks

**Pull request merged**